src/theme/styles/global.ts

```diff
@@ -0,0 +1,112 @@
+import { createGlobalStyle, css } from 'styled-components'
+
+import fontFace from './fonts'
+import { web3ModalOverride } from './overrides'
+
+// TODO: remove for constants from colour palette later
+import variables from 'components/layout/GenericLayout/variablesCss'
+
+// TODO: replace these variables w/Colour constants (later):
+// --color-text-primary: #456483
+// --color-gradient-1: #21222E
+// --color-gradient-2: #2C2D3F
+// --color-background-selection: #218DFF
+// --color-text-active: #218DFF
+
+const selection = css`
+  /* CSS for selecting text */
+  *::selection {
+    background: #218dff; /* WebKit/Blink Browsers */
+  }
+  *::-moz-selection {
+    background: #218dff; /* Gecko Browsers */
+  }
+  *::-webkit-selection {
+    background: #218dff; /* Chrome Browsers */
+  }
+  /* End CSS for selecting text */
+`
+
+const scrollbars = css`
+  ::-webkit-scrollbar {
+    width: 6px !important;
+    height: 6px !important;
+  }
+  ::-webkit-scrollbar-thumb {
+    background-color: rgba(0, 0, 0, 0.2);
+  }
+  ::-webkit-scrollbar-track {
+    background: hsla(0, 0%, 100%, 0.1);
+  }
+`
+
+export const StaticGlobalStyle = createGlobalStyle`
+  /* TEMPORARY: import variables */ 
+  ${variables}
+  
+  /* Import font faces */
+  ${fontFace}
+
+  /* Selection CSS */
+  ${selection}
+
+  /* Scrollbars CSS */
+  ${scrollbars}
+
+  .noScroll {
+    overflow: hidden;
+  }
+
+  .not-implemented {
+    display: none !important
+  }
+
+  html, body {  
+    height: 100vh;
+    width: 100vw;
+    margin: 0;
+    font-size: 62.5%;
+    line-height: 10px;
+    font-family: var(--font-default);
+    background-image: linear-gradient(0deg, var(--color-gradient-1) 0%, var(--color-gradient-2) 100%);
```

Comment:
We can address later, but I would think when you substitute `--color-gradient-1` with colors in the palette, since the palette is theme related, this settings would be theme dependent. 

The point of having a static css, is to have a nice default reset for all themes, right?

All these styles that have to do with color shouldn't be part of `ThemedGlobalStyle` instead?

For reference, look at Uniswap FixedGlobalStyle, it's nice to have some styles that are not re-render when the theme changes: 
![image](https://user-images.githubusercontent.com/2352112/104015318-b9512c80-51b4-11eb-8723-9010bdb83c15.png)

Reply:
yep all this changes later in the chain